docker-compose.yml

```diff
@@ -42,7 +42,7 @@ services:
     # networks:
       # - elastic
   postgres_db:
-    image: postgres:11.12
+    image: postgres:13.3
```

Comment:
for JIT see other 12 PR. 
for 200 connections, the comment is originally 
> Some sort of connection leak (by the internal dropwizard validation query), could not figure out how to fix, just increased db max_connections 
in relation to I think tests, so I think it doesn't apply here @garyluu  can you confirm?

Reply:
> for JIT see other pg 12 PR.

Do you mean other PR turns off JIT, or there's a comment there I should look at?

Reply:
I think I meant this comment https://github.com/dockstore/dockstore/pull/4390#issuecomment-896074280 which was made when I assumed AWS RDS had JIT on. In that case, I didn't decide whether I wanted JIT on or off for Docker compose to match the tests or match prod. 


Since you discovered that prod has JIT off, think I'll turn it off here too